data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -360,6 +360,7 @@
 
         {CLEAR_VARIABLE malin}
 
+        # FIXME: allow_undo like this causes corrupt replays.
         [allow_undo][/allow_undo]
```

Comment:
Why not just remove these since they're clearly bogus?

Reply:
I'm not sure what the correct approach is here:
1) remove the [allow_undo]
2) remove the xp gain
3) make it somehow work with [on_undo]

Reply:
Ah, [on_undo] could be an option, yeah.

Reply:
For this one, I think removing the [allow_undo] is the best choice. The hex it's on is en-route from the starting castle to a village that's perfectly placed for the leader to capture on the first turn, so I'd expect the player to pick this up and then immediately grab the village and end turn.